packages/stickers/src/beauty/cta.js

```diff
@@ -16,8 +16,11 @@
 /**
  * External dependencies
  */
+import { __ } from '@web-stories-wp/i18n';
 import PropTypes from 'prop-types';
 
+const title = __('Heart', 'web-stories');
```

Comment:
Let's use `_x()` everywhere so translators have more context.

```suggestion
const title = _x('Heart', 'sticker name', 'web-stories');
```